packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -447,7 +448,7 @@ async function checkTurboSupportsCorepack(
   const turboJsonPath = path.join(rootDir, 'turbo.json');
   const turboJsonExists = await fs.pathExists(turboJsonPath);
   const turboJson = turboJsonExists
-    ? (JSON.parse(await fs.readFile(turboJsonPath, 'utf8')) as TurboJson)
+    ? (json5.parse(await fs.readFile(turboJsonPath, 'utf8')) as TurboJson)
     : undefined;
   return turboJson?.globalPassThroughEnv?.includes('COREPACK_HOME') || false;
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** Can we add a test for parsing json with comments?‏